README.md

```diff
@@ -17,42 +11,36 @@ ways:
 
 **Better default formatting for CloudWatch notifications:**
 
-![AWS Cloud Notification for Slack](https://github.com/assertible/lambda-cloudwatch-slack/raw/master/images/cloudwatch.png)
+![AWS Cloud Notification for Slack](https://github.com/pixel-point/lambda-cloudwatch-slack/raw/master/images/cloudwatch.png)
```

Comment:
Why were these changed to pixel point?

Reply:
It's mistake. I will revert this changes.